### PR TITLE
 [olivetin] add source/guide_olivetin.rst close #1270

### LINKED ISSUE #1270: [olivetin] add a guide for OliveTin
Looks good and fits well.

https://github.com/OliveTin/OliveTin

https://www.olivetin.app

### PR DESCRIPTION
**Pull request by kimdiallo**

### CODE REVIEW

File: source/guide_olivetin.rst [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+OliveTin is released under the `AGPL-3.0 License`_.
+
+
+Installation
+============
+
+Download OliveTin
+------------------
+
+Check Github_ for the `latest release`_ and copy the download link to the linux-amd64.tar.gz file.
```
Comment (noave) [inline source/guide_olivetin.rst:44]: filename would look better in code ticks :-)

File: source/guide_olivetin.rst [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+
+Extract OliveTin
+--------------------------
+
+Create a directory and unpack the archive into it.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir ~/OliveTin
```
Comment (noave) [inline source/guide_olivetin.rst:59]: i personally like lowercase folder names much more

File: source/guide_olivetin.rst [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+------------------
+
+Create a directory for the config file and copy the ``~/OliveTin/config.yaml`` into it.
+
+.. code-block:: ini
+
+ [isabell@stardust ~]$ mkdir ~/etc/OliveTin
+ [isabell@stardust ~]$ cp ~/OliveTin/config.yaml ~/etc/OliveTin/config.yaml
+
+Now you can add scripts and shell commands in '~/etc/OliveTin/config.yaml' which should be executed via the web interface. Use the examples in the file and the configdoc_ for orientation.
```
Comment (noave) [inline source/guide_olivetin.rst:86]: wrong backticks for codestring

File: source/guide_olivetin.rst [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+
+Use your favourite editor to create ``~/etc/services.d/olivetin.ini`` with the following content:
+
+.. code-block:: ini
+
+ [program:olivetin]
+ directory=%(ENV_HOME)s/OliveTin
+ command=%(ENV_HOME)s/OliveTin/OliveTin --configdir %(ENV_HOME)s/etc/OliveTin
+ autostart=yes
+ autorestart=yes
```
Comment (noave) [inline source/guide_olivetin.rst:99]: add startsecs=30

File: source/guide_olivetin.rst [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+    OliveTin is running on port 1337 in the default configuration. If you want or need to use another port, you can change that in the ``~/OliveTin/config.yaml``..
+
+.. include:: includes/web-backend.rst
+
+
+Updates
+=======
+
+Periodically check the latest release_ page to learn about new versions.
```
Comment (noave) [inline source/guide_olivetin.rst:118]: is there a feed somewhere?

**Review by noave — CHANGES_REQUESTED** [review]
some minor comments in code, generally I would recommend to use lowercase named folder and filenames, its more of a personal preference

File: source/guide_olivetin.rst (hunk 2 above)
Comment (kimdiallo) [reply source/guide_olivetin.rst:59]: The spelling is based on that of the source, so it occurs anyway and even if I see this in itself just like you, I would prefer a uniform spelling in any case.

File: source/guide_olivetin.rst (hunk 5 above)
Comment (kimdiallo) [reply source/guide_olivetin.rst:118]: Maybe, but the release page can be easily checked without a reader, so it is more convenient that way.

File: source/guide_olivetin.rst (hunk 4 above)
Comment (noave) [reply source/guide_olivetin.rst:99]: If you have a better value for startsecs thats fine, but we should just add it at all so starting loops are prevented :-)

File: source/guide_olivetin.rst (hunk 5 above)
Comment (noave) [reply source/guide_olivetin.rst:118]: But you have to go there actively and check for new releases, its better to have a feed reader do so for you, its also part of our style guide:

> Try to find an RSS feed for updates and document it.

**Comment by luto** [comment]
Feel free to reopen, once there as been some progress :)

**Pull request closed without merging**